spec/models/emotion_spec.rb

```diff
@@ -30,12 +30,6 @@
       expect(new_word).to_not be_valid
     end
 
-    it "should lowercase words" do
-      emotion = Emotion.new(word: "Enthusiastic")
-      emotion.save!
-      expect(emotion.word).to include("enthusiastic")
-    end
-
     it "the capitalized emotion word should not be used" do
       new_word = FactoryBot.build(:emotion, word: emotion.word.upcase!)
       expect(new_word).to_not be_valid
```

Comment:
This test does not make sense because there is no validation against upper case words. Instead create a new section to test the `before_save` action:

```ruby
describe 'Before actions' do
  it 'during creates and saves it downcases the emotion.word' do
     feeling = create(:emotion, word: 'HAPPY')
     expect(feeling.reload.word).to eq('happy')
     feeling.update(word: 'Sad')
     expect(feeling.reload.word).to eq('sad')
  end
end
```